src/Microsoft.ML.Transforms/Text/WrappedTextTransformers.cs

```diff
@@ -119,4 +122,614 @@ private static TransformWrapper MakeTransformer(IHostEnvironment env, (string in
             return new TransformWrapper(env, new CharTokenizeTransform(env, args, emptyData));
         }
     }
+
+    /// <summary>
+    /// Stopword remover removes language-specific lists of stop words (most common words)
+    /// This is usually applied after tokenizing text, so it compares individual tokens
+    /// (case-insensitive comparison) to the stopwords.
+    /// </summary>
```

Comment:
I am thinking to move these XML comments into doc.xml.

Reply:
When I put the doc.xml there, initially, it was temporarely, so that the docs can be on the runtime, and the entry point classes. the thought then was to remove them, after the CSharpApi would go. Will take another survey on whether people actually prefer to have the docs.

Reply:
ok, then I keep the comments as-is for now.

---
In reply to: [219248683](https://github.com/dotnet/machinelearning/pull/953#discussion_r219248683) [](ancestors = 219248683)